_episodes/12-cmdline.md

```diff
@@ -22,7 +22,10 @@ In order to do that,
 we need to make our programs work like other Unix command-line tools.
 For example,
 we may want a program that reads a dataset
-and prints the average inflammation per patient.
+and prints the average inflammation per patient. One of our goals is to produce a program which prints
+the average inflammation per patient in a given file using command line arguments and flags. 
+The second goal is to use shell commands and flags to look at a minimum of the first four lines. 
+A third goal is to find the maximum inflammation in different inflammation files.
```

Comment:
I tend to think of these programs/commands listed in the text you provide here as examples that lead us to our goals (rather than the goals themselves) which I consider to be the Objectives listed in the episode Overview. Between what is given in the Objectives section and the text given before each of the examples (i.e. "prints the average inflammation per patient for a given file", "might also want to look at the minimum of the first four lines", "or the maximum inflammations in several files one after another"), how do you see this change expanding on what was there previously?

Reply:
I think my initial confusion came from jumping right into using the command line with flags when the lesson hadn't started yet. It was abrupt and didn't flow, there was no introduction. There may be a better way to address this without restating the goals, but I can't tell you what that is.

Reply:
I see what you mean. When taught in a workshop, this lesson usually comes after the Unix Shell lesson where learners see commands with arguments and flags, which maybe makes this less jarring for some? Perhaps to help ease into the appearance of arguments and flags in the episode, we could modify the first paragraph to include something about them by adding to this sentence:

> In order to do that, we need to make our programs work like other Unix command-line tools, allowing us to use command-line arguments and flags to supply values used in the program execution.

or maybe for another possibility, add the second sentence here:
> In order to do that, we need to make our programs work like other Unix command-line tools. Let's look at how we can use command-line arguments and flags to supply values to our programs.

Do you think this would help clarify things somewhat?